getinvolved.html

```diff
@@ -5,9 +5,9 @@
 <h1 class="openlex-header">Get Involved with Open Lexington:</h1>
 <div class="getinvolved">
   <ul class="getinvolved">
-  <li id="getinvolved"><a href="https://groups.google.com/forum/#!forum/openlexington" class="project-title">Join the discussion on GoogleGroups</a></li>
-  <li><a href="https://twitter.com/openlexington/" class="project-title">Follow us on Twitter</a></li>
-  <li><a href="https://github.com/openlexington" class="project-title">Fork our code on GitHub</a></li>
-  <li><a href="http://meetup.com/OpenLexington" class="project-title">Attend a Hack Event</a></li>
+  <li id="getinvolved"><a href="https://groups.google.com/forum/#!forum/openlexington" class="project-title" target="_blank">Join the discussion on GoogleGroups</a></li>
+  <li><a href="https://twitter.com/openlexington/" class="project-title" target="_blank">Follow us on Twitter</a></li>
+  <li><a href="https://github.com/openlexington" class="project-title" target="_blank">Fork our code on GitHub</a></li>
+  <li><a href="http://meetup.com/OpenLexington" class="project-title" target="_blank">Attend a Hack Event</a></li>
```

Comment:
we may have to come back around to this one... since our budget is gone, we can't afford to continue using meetup.com